sdk/spring/CHANGELOG.md

```diff
@@ -26,14 +26,36 @@ This section includes changes in `spring-cloud-azure-stream-binder-eventhubs` mo
 #### Breaking Changes
 - Change the type of the binding producer property of `send-timeout` from `long` to `Duration` [#26625](https://github.com/Azure/azure-sdk-for-java/pull/26625).
 
-### Other Changes
-
 ### Spring Cloud Azure Event Hubs Starter
 This section includes changes in `spring-cloud-azure-starter-eventhubs` module.
 
 #### Breaking Changes
 - Remove property of `spring.cloud.azure.eventhubs.processor.partition-ownership-expiration-interval` which can be replaced by
 `spring.cloud.azure.eventhubs.processor.load-balancing.partition-ownership-expiration-interval` [#27331](https://github.com/Azure/azure-sdk-for-java/pull/27331).
+
+### Spring Messaging Azure
+
+#### Breaking Changes
+- Move class `com.azure.spring.messaging.PartitionSupplier` to library com.azure.spring:spring-messaging-azure-eventhubs, which is `com.azure.spring.eventhubs.core.PartitionSupplier` [#27422](https://github.com/Azure/azure-sdk-for-java/issues/27422).
```

Comment:
Moary has changed the package name to `com.azure.spring.messaging.eventhubs`. We'd better merge the latest 4.0 branch and also change this `com.azure.spring.eventhubs.core.PartitionSupplier` to `com.azure.spring.messaging.eventhubs.core.PartitionSupplier`.